The lint throws me an error in a line not related with this PR:
```
lint via golangci-lint
pkg/services/alerting/notifiers/googlechat.go:59: File is not `gofmt`-ed with `-s` (gofmt)
	Cards []card `json:"cards"`
make: *** [Makefile:95: golangci-lint] Error 1
```
Could you help me? Thanks in advance.

--- a/pkg/services/alerting/notifiers/googlechat.go
+++ b/pkg/services/alerting/notifiers/googlechat.go
@@ -55,7 +55,8 @@ Structs used to build a custom Google Hangouts Chat message card.
 See: https://developers.google.com/hangouts/chat/reference/message-formats/cards
 */
 type outerStruct struct {
-	Cards []card `json:"cards"`
+	FallbackText string `json:"fallbackText"`
+	Cards []card        `json:"cards"`
 }
 
 type card struct {
@@ -187,6 +188,7 @@ func (gcn *GoogleChatNotifier) Notify(evalContext *alerting.EvalContext) error {
 
 	// nest the required structs
 	res1D := &outerStruct{
+		FallbackText: evalContext.GetNotificationTitle(),
 		Cards: []card{
 			{
 				Header: header{

--- a/pkg/services/alerting/notifiers/googlechat.go
+++ b/pkg/services/alerting/notifiers/googlechat.go
@@ -56,7 +56,7 @@ See: https://developers.google.com/hangouts/chat/reference/message-formats/cards
 */
 type outerStruct struct {
 	FallbackText string `json:"fallbackText"`
-	Cards []card        `json:"cards"`
+	Cards        []card `json:"cards"`
 }
 
 type card struct {

